tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -496,6 +496,9 @@ public function testEventRendering(): void {
     $this->assertSession()->linkNotExistsExact('Main link for media items');
     $this->assertSession()->pageTextNotContains('More media links');
 
+    // Assert "Description" title is not rendered unless there is a body text.
+    $this->assertSession()->pageTextNotContains('Description');
+
     // Assert "Full text", "Featured media", "Featured media legend" fields
```

Comment:
Probably tests should be slightly updated because the following behavior is deprecated.